src/styles/blocks/body.scss

```diff
@@ -0,0 +1,5 @@
+.body {
+  &__calendar {
+    margin: 50px auto 0;
```

Comment:
I think you can do this direclty for `calendar` block

Reply:
Is it possible to set an external geometry (margin) for a block?